packages/main/src/DatePicker.hbs

```diff
@@ -46,6 +51,6 @@
 				@ui5-selectedDatesChange="{{_calendar.onSelectedDatesChange}}"
 		></ui5-calendar>
 	</ui5-popover>
-
+	<span id="{{_id}}-date" class="ui5-hidden-text">{{dateAriaDescription}}</span>
```

Comment:
this is not being read out when it is outside the native input DOM. We should add it in the same shadow root as the native input element